snapshot/pkg/volume/openebs/processor.go

```diff
@@ -69,15 +78,22 @@ func (h *openEBSPlugin) SnapshotCreate(snapshot *crdv1.VolumeSnapshot, pv *v1.Pe
 
 	snapObj := (*tags)["kubernetes.io/created-for/snapshot/name"]
 	snapshotName := createSnapshotName(pv.Name, snapObj)
+
 	casType := pv.Annotations["openebs.io/cas-type"]
 	if casType == "" {
 		casType = "jiva"
 	}
+
+	ok := supportedCasType[casType]
```

Comment:
q_: what is the value returned when the cas-type is not in the array. is it nil or false?

Reply:
it returns `false` as false is the default value of boolean

Reply:
for example https://play.golang.org/p/atymMpD1i3d